CMakeLists.txt

```diff
@@ -683,6 +683,16 @@ if(BUILD_HELICS_TESTS AND BUILD_TESTING AND ENABLE_TEST_CORE)
     add_subdirectory(tests)
 endif()
 
+option(
+  HELICS_BUILD_BENCHMARKS
```

Comment:
This option seems similar to the one for building tests -- could we rename it to `BUILD_HELICS_BENCHMARKS` so CMake groups the options together?

Reply:
Eventually I want to get all the helics cmake options to look like HELICS_XXXX so they are "namespaced"  I don't think we will be able to do that completely until HELICS 3.0 but any new option I have been trying to put into that form.  I suppose I could change it the BUILD_HELICS form again.    But not sure how that would impact things later on or if it doesn't matter much.

Reply:
I am going to merge this now, and I have another branch working with some of the cmake variables that should come in the near future.  if we decide we want to change the variable to BUILD_HELICS_BENCHMARKS we can do it then before the next release.